GenderPayGap.WebUI/Views/AdminOrganisationReturn/ViewReturnDetails.cshtml

```diff
@@ -307,9 +307,9 @@
                                           (thisVersion.FemaleUpperPayBand != previousVersion.FemaleUpperPayBand));
 
                         <td class="govuk-table__cell @(isChanged ? "value-changed-from-previous-return" : null)">
-                            <span style="white-space: nowrap;">@(thisVersion.MaleUpperPayBand.ToString("0"))% male</span>
+                            <span style="white-space: nowrap;">@(thisVersion.MaleUpperPayBand?.ToString("0"))% male</span>

```

Comment:
Should we not display the table at all if there's no values?

Reply:
So this table is used for displaying all the versions of a report and it includes all the figures (not only the pay quarters but also the bonuses and the mean, median figures).

Reply:
Will this table sometimes show an empty cell with just the word male in it though?

Reply:
Yes, it's the same for the bonus figures that are already optional - they have "%" displayed. We can definitely change that but I don't think it really matters as it's an admin table.

We might want to add a description (eg "Employer opted out of reporting pay quarters") but I will check that with GEO and we can make the changes as part of the other ticket when we update the other copies as well.

Thanks for flagging this!

Reply:
Great, that makes sense. Thanks Malina 😄.